message/retrylater.go

```diff
@@ -2,14 +2,15 @@ package message
 
 import (
 	"context"
-	"fmt"
 	"time"
 
 	servicebus "github.com/Azure/azure-service-bus-go"
+	"github.com/devigned/tab"
 )
 
-// RetryLater waits for the given duration before retrying the processing of the message.
-// This happens in memory and does not impact servicebus message max retry limit
+// RetryLater waits for the given duration before abanoning.
+// Consider examining/increasing your LockDuration/MaxDeliveryCount before using.
+// Can't retryly later longer than the lock duration (which has a max of 5 minutes)
```

Comment:
nit: Change to something like "Undefined behavior if RetryLater is passed a duration that puts the message handling past the lock duration (which has a max of 5 minutes)"

If you like the suggestion:
```suggestion
// Undefined behavior if RetryLater is passed a duration that puts the message handling past the lock duration (which has a max of 5 minutes)
```